README.md

```diff
@@ -55,19 +55,24 @@ If you would like to receive updates, follow us on [Bluesky](https://bsky.app/pr
 If you have further questions or suggestions for collaborations, don't hesitate to contact us directly at **info@thousandbrains.org**.
 
 # Citing the Project
-If you're writing a publication that references the Thousand Brains Project, please cite our [TBP white paper](https://arxiv.org/abs/2412.18354):
+If you would like to refer to Monty's capabilities and advantages over deep learning, please cite this pre-print:
+
+[Thousand-Brains Systems: Sensorimotor Intelligence for Rapid, Robust Learning and Inference](https://arxiv.org/abs/2507.04494):
 ```
-@misc{thousandbrainsproject2024,
```

Comment:
Why did you delete this reference? I think it is still a good way for people to cite the project in general (the more recent paper is more about the capabilities of Monty, not the thousand brains project)